src/resources/assets/js/components/dismissImageGridImage.vue

```diff
@@ -43,6 +62,41 @@ export default {
         title() {
             return this.selected ? 'Undo dismissing this annotation' : 'Dismiss this annotation';
         },
+        svg() {
+            return this.xml !== null ? new XMLSerializer().serializeToString(this.xml) : '';
+        },
+    },
+    methods: {
+        svgSrcUrl() {
+            return this.srcUrl.replace(/.[A-Za-z]*$/, '.svg');
+        },
+        async fetchSVG() {
+            await fetch(this.svgSrcUrl()).then(res => res.text()).then(data => {
+                this.setXML(data);
+                // prevent stroke to scale
+                this.addAttributeToShape("vector-effect", "non-scaling-stroke");
```

Comment:
Could you also add this directly when the SVG is generated in the backend? Thes would make handling in the frontend easier (no need to parse the XML and convert it back into a string).